docker/sklearn/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM demisto/python3:3.7.5.5420
```

Comment:
Why not use our latest image based on python 3.8.3?

Reply:
The model is trained using python3.7. Using python 3.8 can cause come mistakes. @glicht